website_sale_checkout_skip_payment/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+==================================
+Website Sale Checkout Skip Payment
+==================================
+
+This module allows to logged users to checkout with no payment step. At the
+end of the checkout proccess the quotation is sent to the user email address
+and set to *Qoutation Sent* state.
+
```

Comment:
Add: "The quotation created in the website_sale will be the same as if it would be created in backend, so all fields from the partner will be in the sale order as "pricelist" "payment term" "sales person", etc..."